### ISSUE TITLE
about the train output feature

### ISSUE THREAD

**Issue by Usernamezhx** [lyakaap/ISC21-Descriptor-Track-1st#5]
sorry to bother you again. I want train the model with a small backbone such as resnet50. Because I only have three GPU and I run with command:
```python
CUDA_VISIBLE_DEVICES=0,1,2 python v83.py  --dist-url 'tcp://localhost:10001' --multiprocessing-distributed --world-size 1 --rank 0 --seed 9 \
  --epochs 5 --lr 0.1 --wd 1e-6 --batch-size 96 --ncrops 2 \
  --gem-p 1.0 --pos-margin 0.0 --neg-margin 1.0 \
  --input-size 256 --sample-size 1000000 --memory-size 20000 \
/root/zhx3/data/fb_train_data/train
```
I find a strange problem. I test checkpoint_000{0..4}.pth.tar model. only the checkpoint_0002.pth.tar ouput different when the input is different. I mean other model will output same embedding no matter what different you input. thanks in advance.
the loss log output such as: 
```python
epoch 5:   0%|          | 0/15873 [00:00<?, ?it/s]=> loading checkpoint './v83/train/checkpoint_0004.pth.tar'
=> loaded checkpoint './v83/train/checkpoint_0004.pth.tar' (epoch 5)
epoch 6:   0%|          | 0/15873 [00:00<?, ?it/s]epoch=5, loss=1.0154363534772417
epoch 7:   0%|          | 0/15873 [00:00<?, ?it/s]epoch=6, loss=1.012835873522891
```

**Comment by Usernamezhx** [comment]
I solved by change the lr to 0.05. Because I only have three GPUs. so I set batch size 64. and I should run more epoch and set a small learning rate. so amazing........
